crates/transaction-pool/src/validate/eth.rs

```diff
@@ -257,6 +268,19 @@ where
 
         // intrinsic gas checks
         let is_shanghai = self.fork_tracker.is_shanghai_activated();
+
+        // 7702 check
+        // TODO: get num auth items
```

Comment:
should this be validated here? Or just on decoding?

Reply:
I'll solve this in a followup, tracking: https://github.com/paradigmxyz/reth/issues/9428